toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -412,21 +412,8 @@ func (r *RpmRepoCloner) WhatProvides(pkgVer *pkgjson.PackageVer) (packageNames [
 }
 
 // ConvertDownloadedPackagesIntoRepo initializes the downloaded RPMs into an RPM repository.
+// Packages will be placed in a flat directory.
 func (r *RpmRepoCloner) ConvertDownloadedPackagesIntoRepo() (err error) {
-	srcDir := filepath.Join(r.chroot.RootDir(), chrootDownloadDir)
-	repoDir := srcDir
-
-	if !buildpipeline.IsRegularBuild() {
-		// Docker based build doesn't use overlay so repo folder
-		// must be explicitely set to the RPMs cache folder
-		repoDir = filepath.Join(r.chroot.RootDir(), cacheRepoDir)
-	}
-
-	err = rpmrepomanager.OrganizePackagesByArch(srcDir, repoDir)
```

Comment:
This is the only place `OrganizePackagesByArch` is used. We can remove that function as well now. No need to keep it, if we're not using it.

Reply:
Fair point, I've cleaned it up.